bot.py

```
@@ -729,7 +733,60 @@ def sort_key(item):
+            sha = commit.get('sha', '')[:7]
+            message = commit.get('commit', {}).get('message', 'No commit message').split('\n')[0]
+            author = commit.get('commit', {}).get('author', {}).get('name', 'N/A')
+            commit_url = commit.get('html_url', '#')
+            
+            # Format: `[sha_link](url)` - `Message` (Author)
+            description_lines.append(
+                f"[`{sha}`]({commit_url}) - `{message}` ({author})"
+            )
+
+        embed.description = "\n".join(description_lines)
+        embed.set_footer(text=f"Showing the 5 most recent commits")
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)